core/domain/html_validation_service_test.py

```diff
@@ -484,7 +484,7 @@ def test_validate_soup_for_rte(self):
 
         for index, test_case in enumerate(test_cases_for_textangular):
             actual_output_for_textangular = (
-                html_validation_service._validate_soup_for_rte( # pylint: disable=protected-access
+                html_validation_service.validate_soup_for_rte( # pylint: disable=protected-access
```

Comment:
Shouldn't you be also removing the pylint-disable, here and below?

Reply:
Ah yes! Thank you for pointing that out. I will make the change.